research-app/src/App.vue

```diff
@@ -1066,7 +1122,7 @@ body {
 #tools {
   position: absolute;
   z-index: 10;
-  bottom: 3rem;
+  top: 4rem;
```

Comment:
I think it might be good to give this a top margin of `0.5rem`, or whatever the separation is for the coordinate readout and hamburger menu? Then everything will be nicely aligned at the top of the UI.